uint8 CliffState
---
bool result